### ISSUE TITLE
Atypon/RIS: Tab characters in title field

### ISSUE THREAD

**Issue by dstillman** [zotero/translators#1955]
I'm creating this here, but we probably want a more general solution.

The RIS served on https://econtent.hogrefe.com/doi/10.1027/1864-9335/a000248 has some embedded tag characters that end up getting saved. (They don't show up the way Zotero currently renders title fields in the item pane, but they show up in the bibliography entry or if you go to edit the field.)

Do we want to just replace all internal spacing in all fields with single spaces in the translator architecture?

**Comment by zuphilip** [comment]
I searched in my library for entries with two spaces in the title and all of them could IMO be replaced by one space. Twitter translator will do that, when you have a tweet with a line break (the tweet itself will become the title).

> Do we want to just replace all internal spacing in all fields with single spaces in the translator architecture?

I would say that would be good for all title fields (title, publicationTitle, seriesTitle etc.). But I wouldn't do this in the abstract field or notes, because there tabs and line breaks may be not uncommon. I don't see much of a case for all the other fields, e.g. it is hard to think about, how this could occur for the creators field and what to do then.

**Comment by dstillman** [comment]
Yeah, I think that's right — I also can't think of any reason you'd need multiple internal spaces for fields other than Abstract and Extra.

**Comment by zuphilip** [comment]
(FYI Some translators also use `ZU.trimInternal` function to fix such multiple spaces.)
